frontend/app/types/cdcp.d.ts

```diff
@@ -0,0 +1,5 @@
+import { type RouteHandle as RemixRouteHandle } from '@remix-run/react/dist/routeModules';
+
+import { type Namespace } from 'i18next';
+
+export type RouteHandle = RemixRouteHandle & { i18nNamespaces: Namespace };
```

Comment:
I don't know if this makes sense. The data in a route handle depends on the needs of the route.

Not all routes will need an i18n namespace (ie: API routes), and some routes may need to emit different optional data (ie: breadcrumb data).

Reply:
Fixed! :)